src/linker/Linker/LinkContext.cs

```diff
@@ -120,6 +120,10 @@ public bool IgnoreUnresolved
 
 		public List<string> Substitutions { get; private set; }
 
+		public HashSet<PInvokeInfo> PInvokes { get; private set; }
```

Comment:
It should be just List<PInvokeInfo> after expanding PInvokeInfo data to be always unique